### PR TITLE
Replace git.maze.io/go/duration to github.com/hako/durafmt

### PR DESCRIPTION
**Pull request by PuneetPunamiya**
- Current package we used to compute the duration is broken,
so this patch adds an alternative dependency to compute
the duration for the token

- This patch adds a note in deployment docs to specify
the formats supported by `ACCESS_JWT_EXPIRES_IN` and
`REFRESH_JWT_EXPIRES_IN`

Signed-off-by: Puneet Punamiya <ppunamiy@redhat.com>

# Changes

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [tests](https://github.com/tektoncd/community/blob/main/standards.md#principles) (if functionality changed/added)
- [ ] Run API Unit Tests, Lint Checks, API Design, Golden Files with `make api-check`
- [ ] Run UI Unit Tests, Lint Checks with `make ui-check`
- [ ] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/main/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/pipeline/blob/main/CONTRIBUTING.md) for more details._

### CODE REVIEW

File: api/pkg/app/app.go [hunk 1]
```diff
@@ -401,9 +402,25 @@ func jwtConfig() (*JWTConfig, error) {
...
+		if err != nil {
+			return 0, err
+		}
+		dur = fmt.Sprintf("%dh", (weeks * 7 * 24))
+	}
+
+	d, err := durafmt.ParseString(dur)
 	if err != nil {
-		return 0, err
+		return 0, fmt.Errorf("JWT doesnn't support the duration specified %s. \nSupported formats are w(weeks), d(days), h(hours), m(min), s(sec)", dur)
```
Comment (sm43) [inline api/pkg/app/app.go:423]:
```suggestion
		return 0, fmt.Errorf("JWT doesn't support the duration specified %s. \nSupported formats are w(weeks), d(days), h(hours), m(min), s(sec)", dur)
```

**Review by vdemeester — APPROVED** [review]

**Comment by piyush-garg** [comment]
Can we add a small note about the limitation in docs?

File: api/pkg/app/app_test.go [hunk 2]
```diff
@@ -0,0 +1,20 @@
+package app
```
Comment (vinamra28) [inline api/pkg/app/app_test.go:1]: can we add a license here?

**Review by vinamra28 — APPROVED** [review]

**Pull request merged**